pyfranca/franca_lexer.py

```diff
@@ -144,6 +175,28 @@ def t_INTEGER(t):
         t.value = int(t.value)
         return t
 
+
+    # noinspection PyPep8Naming,PyIncorrectDocstring
+    @staticmethod
+    def t_BOOLEAN_VAL(t):
+        # noinspection PySingleQuotedDocstring
+        r"(true|false)"
+        t.value = t.value.strip()
+        if t.value == "true":
+            t.value = True
+        else:
+            t.value = False
+        return t
+
+    # Identifier
```

Comment:
yes it is. I was suprised to see that the order of evalutaing the regular expression is not the order in array token[]. It is the order of implementation of all the static methods, e.g. t_BLOCK_COMMENT etc. Thats the reason why I have to first implement t_FLOAT_VAL before t_INTEGER. Otherwiese FLOAT_VAL never matches.